.github/CONTRIBUTING.md

```diff
@@ -142,7 +156,9 @@ builder failures, you may be experiencing a [test suite
 issue](https://github.com/zfsonlinux/zfs/issues?q=is%3Aissue+is%3Aopen+label%3A%22Test+Suite%22).
 There are also various [buildbot options](https://github.com/zfsonlinux/zfs/wiki/Buildbot-Options)
 to control how changes are tested.
-* All proposed changes must be approved by a ZFS on Linux organization member.
+* If you have an idea you'd like to discuss or which requires additional testing consider opening it as a draft pull request. 
+Once everything is in good shape and the details have been worked out you can remove its draft status. 
+Any required reviews can then be finalized and the pull request merged.
```

Comment:
Let's move this bullet to the end of the "General" section.  After the rewording I think it would make the most sense there.

Reply:
Good point, it has nothing (directly) to do with testing anymore...